Gemfile

```diff
@@ -20,3 +20,4 @@ gem "tapioca", "~> 0.13", require: false, platforms: :ruby
 gem "psych", "~> 5.1", require: false
 gem "rails"
 gem "webmock"
+gem "tzinfo-data"
```

Comment:
Needed for Windows platforms. Trying to limit the platforms was causing some Bundler issues, so I've left it open.

Reply:
What issues?